rskj/6.3.0-arrowhead/README.md

```diff
@@ -0,0 +1,35 @@
+# rskj ARROWHEAD-6.3.0
+
+* Source: https://github.com/rsksmart/rskj
+* Tag: `ARROWHEAD-6.3.0`
+
+## Build
+
+```
+$ docker build -t rskj/6.3.0-arrowhead .
+```
+
+## Verify
+
+The last step of the build prints the sha256sum of the files, if, for any reason there's a need to recheck the hash the following commands can be used to generate them.
+
+```
+$ docker run --rm rskj/6.3.0-arrowhead sh -c 'sha256sum * | grep -v javadoc.jar'
```

Comment:
```suggestion
$ docker run --rm rskj/6.3.0-arrowhead sh -c 'sha256sum *'
```
By adding the changes I suggested in the Dockerfile no need to skip javadoc anymore. Just an idea